### ISSUE TITLE
Magisk Manager Crashes when on the Log's Screen

### ISSUE THREAD

**Issue by lnx64** [topjohnwu/Magisk#1734]
Phone Specs:
Asus Zenfone 4 Max (X00HD / ZC520KL)
Android Pie AOSP (Official Beta from Asus, which is actually more stable than ZenUI Oreo)
Latest version of Magisk and Manager.

When viewing the Log's screen, Magisk Manager immediately crashes.  I am unable to view logs as a result.  Attempting to clear Data and Cache from the Manager doesn't help.  Attached is the version numbers to confirm, and the modules currently installed.

![bsTTYhD](https://user-images.githubusercontent.com/2136976/62323019-5a8e6c00-b474-11e9-8d8d-b34c31e286e3.png)
![o6jv1tG](https://user-images.githubusercontent.com/2136976/62323024-5cf0c600-b474-11e9-9ee2-9bd9bb4f6a3e.png)

**Comment by DenyDarko** [comment]
Logcat is what you need to upload

**Comment by lnx64** [comment]
Apologies, here you go.

[Log_2019-08-01_17-45-30.txt](https://github.com/topjohnwu/Magisk/files/3458769/Log_2019-08-01_17-45-30.txt)

**Comment by lnx64** [comment]
Per app log.
[Log_2019-08-01_17-50-17.txt](https://github.com/topjohnwu/Magisk/files/3458776/Log_2019-08-01_17-50-17.txt)

**Comment by topjohnwu** [comment]
From the log the error does not happen in any of my code, so either support library is broken, or your ROM is broken. Since you are using a beta ROM, I am 99% sure it is the latter case

**Comment by lnx64** [comment]
Alright.  I wasn't sure what the root cause was.  I did end up solving it by forcefully clearing the logs manually outside of Magisk Manager (since it couldn't open it to clear it) and since I did that, now it's working, so idk if the log was malformed and causing issues or what, but now it loads.  Thanks anyways.
![kHjPJST](https://user-images.githubusercontent.com/2136976/62385692-c1b72980-b523-11e9-881d-daa617646750.png)
